code/modules/mob/living/simple_animal/abnormality/teth/penitent_girl.dm

```diff
@@ -25,38 +26,83 @@
 	gift_type =  /datum/ego_gifts/sorrow
 	abnormality_origin = ABNORMALITY_ORIGIN_WONDERLAB
 
-
+//Work Mechanics
 /mob/living/simple_animal/hostile/abnormality/penitentgirl/AttemptWork(mob/living/carbon/human/user, work_type)
-	//Temp too high, random damage type time.
-	if(get_attribute_level(user, TEMPERANCE_ATTRIBUTE) >= 40)
+	//Prudence too high, random damage type time.
+	if(get_attribute_level(user, PRUDENCE_ATTRIBUTE) >= 40)
 		work_damage_type = pick(WHITE_DAMAGE, RED_DAMAGE, BLACK_DAMAGE, PALE_DAMAGE)
 	return TRUE
 
 /mob/living/simple_animal/hostile/abnormality/penitentgirl/PostWorkEffect(mob/living/carbon/human/user, work_type, pe, work_time)
 	// you are going to cut your own leg off
-	if(get_attribute_level(user, TEMPERANCE_ATTRIBUTE) < 40)
-		user.apply_status_effect(STATUS_EFFECT_PENITENCE)
-		to_chat(user, "<span class='danger'>Something feels strange.</span>") //I have to write something here
+	if((get_attribute_level(user, TEMPERANCE_ATTRIBUTE) < 40) && (get_attribute_level(user, PRUDENCE_ATTRIBUTE) < 40))
+		user.apply_damage(250, WHITE_DAMAGE, null, user.run_armor_check(null, WHITE_DAMAGE), spread_damage = TRUE)//DIE!
 		work_damage_type = initial(work_damage_type)
```

Comment:
Should be moved up, outside of condition, to make sure it always has correct damage type for people with low prudence.

Reply:
This is just meant to panic the user, not play into the usual work mechanic.